packages/map-template/src/components/MapTemplate/MapTemplate.jsx

```diff
@@ -556,6 +558,21 @@ function MapTemplate({ apiKey, gmApiKey, mapboxAccessToken, venue, locationId, p
         setCategories(uniqueCategories);
     }
 
+    /*
+     * React on changes in the category prop.
+     * Check if the category property matches with any of the existing categories.
+     */
+    useEffect(() => {
+        if (mapsindoorsSDKAvailable) {
+            if (category) {
+                const existingCategory = categories.find((matched) => matched[0] === category);
+                if (existingCategory !== undefined) {
+                    setSelectedCategory(category);
+                }
+            }
+        }
```

Comment:
Can it be shortened to (and please test 😅 )

```suggestion
        if (mapsindoorsSDKAvailable && category && categories.find((matched) => matched[0] === category)) {
            setSelectedCategory(category);
        }
```

Reply:
Yess, it seems to be working in the shortened version. ✅

Reply:
Pushed the changes from my local branch.